doc/syntax.page

```diff
@@ -423,6 +423,14 @@ of the code block to the end of the starting line:
     end
     ~~~
 
+You can also use "-" in the language name:
+
+    ~~~ ruby-trunk
+    def what?
+      42
+    end
+    ~~~
+
 
```

Comment:
Please only add an example that uses a valid, real world language. Otherwise people might think that `ruby-trunk` is some kind of Ruby based language.

Personally, I would not add any example because we never explicitly state the syntax requirements for the language. Also if a user wants to highlight a programming language that contains a dash he will just use it in the fenced code blocks.